src/app/components/dashboard/dashboard.component.html

```diff
@@ -1,4 +1,40 @@
 <div class="dashboard__content">
+  <div *ngIf="isLoggedIn()">
+    <p>Welcome, {{ userInfo?.info?.name }}</p>
```

Comment:
If you're logged in you automatically get access to name and email ? Are all those "?" necessary ?